src/admin/src/components/page/advisory/Advisory.js

```diff
@@ -494,7 +487,10 @@ export default function Advisory({
             }
             return result;
           });
-          const advisoryStatuses = tempAdvisoryStatuses.filter(
+          const sortedStatus = tempAdvisoryStatuses.sort(
```

Comment:
This line is dying on my local because there are 2 nulls in the tempAdvisoryStatuses array and a.code/b.code are causing errors